README.adoc

```diff
@@ -5,3 +5,12 @@ NOTE: Extends link:https://github.com/jboss-container-images/jboss-openjdk-image
 # License
 
 See link:LICENSE[LICENSE] file.
+
+# TODO
+
+- [ ] Enable GitHub issue to allow people to give feedback
```

Comment:
Hey @rhuss - as @jmtd said - this is a product image - we're tracking product issues in CLOUD jira: https://issues.jboss.org/browse/CLOUD.